GVFS/GVFS/CommandLine/DehydrateVerb.cs

```diff
@@ -260,9 +264,10 @@ private void DehydrateFolders(JsonTracer tracer, GVFSEnlistment enlistment, stri
                                     {
                                         if (!this.TryIO(tracer, () => this.fileSystem.DeleteDirectory(fullPath), $"Deleting '{fullPath}'", out ioError))
                                         {
-                                            this.WriteMessage(tracer, $"Cannot dehydrate folder '{folder}': removing '{folder}' failed.");
+                                            this.WriteMessage(tracer, $"Cannot {this.DisplayName} folder '{folder}': removing '{folder}' failed.");
                                             this.WriteMessage(tracer, "Ensure no applications are accessing the folder and retry.");
                                             this.WriteMessage(tracer, $"More details: {ioError}");
+                                            folderErrors.Add($"{folder}\0{ioError}");

```

Comment:
Looks like a space
```
Scripts The process cannot access the file 'C:\gvfs_repos\ft\src\Scripts' because it is being used by another process.
Failed to dehydrate 1 folder(s).
Failed to prune. Exit Code: DehydrateFolderFailures
```